sevenseconds/config/vpc.py

```diff
@@ -622,6 +623,100 @@ def create_vpc_endpoints(account: AccountData, vpc: object, region: str):
                             ).encode('utf-8')).hexdigest()
                     }
                     act.warning('missing, make create: {}'.format(ec2c.create_vpc_endpoint(**options)))
+        elif service_name.endswith('.kms'):
+            with ActionOnExit('Checking VPC Endpoint {}..'.format(service_name)) as act:
+                sg_name = 'KMS VPC Endpoint'
+                sg_desc = 'Allow access to the KMS VPC endpoint'
+                sg = get_sg(sg_name, sg_desc, vpc.security_groups.all())
+                if not sg:
+                    sg = vpc.create_security_group(
+                        GroupName=sg_name,
+                        Description=sg_desc,
+                    )
+                    time.sleep(2)
```

Comment:
Why?

Reply:
https://github.com/zalando-stups/sevenseconds/pull/102/files#diff-38fa97eea06ec87d9341068431e488cf1b2a9bce65368987532ec3cd525e088fL296
not sure if needed